### PR TITLE
Lock entity by entity in order.

### PR DESCRIPTION
**Pull request by geekbeast**

### CODE REVIEW

File: src/main/kotlin/com/openlattice/data/storage/PostgresDataQueries.kt [hunk 1]
```diff
@@ -354,8 +354,8 @@ fun buildUpsertEntitiesAndLinkedData(): String {
  */
 val lockEntitiesInIdsTable =
         "SELECT 1 FROM ${IDS.name} " +
-                "WHERE ${ENTITY_SET_ID.name} = ? AND ${ID_VALUE.name} = ANY(?) AND ${PARTITION.name} = ? " +
-                "ORDER BY ${ID_VALUE.name} " +
+                "WHERE ${ENTITY_SET_ID.name} = ? AND ${ID_VALUE.name} = ? AND ${PARTITION.name} = ? " +
+//                "ORDER BY ${ID_VALUE.name} " +
```
Comment (katherinebernstein) [inline src/main/kotlin/com/openlattice/data/storage/PostgresDataQueries.kt:358]: is this supposed to be commented out?

Comment (katherinebernstein) [reply src/main/kotlin/com/openlattice/data/storage/PostgresDataQueries.kt:358]: oh i see

Comment (geekbeast) [reply src/main/kotlin/com/openlattice/data/storage/PostgresDataQueries.kt]: Yes, but I should actually just remove the comment.

Comment (katherinebernstein) [reply src/main/kotlin/com/openlattice/data/storage/PostgresDataQueries.kt]: How will the case where there are two batch writes at the same time writing to overlapping sets of entities work? (ie if write1 locks EKID1, write2 locks EKID2, write1 tries to lock EKID2, write2 tries to lock EKID1)

Comment (katherinebernstein) [reply src/main/kotlin/com/openlattice/data/storage/PostgresDataQueries.kt]: ah i guess the sorted() should take care of this

**Pull request merged**